update/common/scripts/5.4/cleanuntranslatablerelations.php

```diff
@@ -53,12 +53,17 @@
  */
 function parseRelationListIds( $relationListXml )
 {
+    if ( empty( trim( $relationListXml ) ) )
+        return;
```

Comment:
Yes, it should return array for consistency. (the `empty()` check on the function result would still skip it anyway)
updating, thanks!